pocs/focuser/birger.py

```diff
@@ -403,33 +403,42 @@ def _initialise(self):
     def _get_serial_number(self):
         response = self._send_command('sn', response_length=1)
         self._serial_number = response[0].rstrip()
-        self.logger.debug("Got serial number {} for {} on {}".format(self.uid, self.name, self.port))
+        self.logger.debug("Got serial number {} for {} on {}".format(
+            self.uid,
+            self.name,
+            self.port))
 
     def _get_library_version(self):
         response = self._send_command('lv', response_length=1)
         self._library_version = response[0].rstrip()
-        self.logger.debug("Got library version '{}' for {} on {}".format(self.library_version, self.name, self.port))
+        self.logger.debug("Got library version '{}' for {} on {}".format(self._library_version,
+                                                                         self.name,
+                                                                         self.port))
 
     def _get_hardware_version(self):
         response = self._send_command('hv', response_length=1)
         self._hardware_version = response[0].rstrip()
-        self.logger.debug("Got hardware version {} for {} on {}".format(self.hardware_version, self.name, self.port))
+        self.logger.debug("Got hardware version {} for {} on {}".format(self._hardware_version,
+                                                                        self.name,
+                                                                        self.port))
 
     def _get_lens_info(self):
         response = self._send_command('id', response_length=1)
```

Comment:
I'm certainly seeing a pattern here w.r.t. send_command, the response length, the element of the response selected and rstrip. While cleaning up, perhaps a helper method could be added to encapsulate the pattern?

Reply:
I almost added that in. I was trying to not do too much change but instead just pull the items from #574 not related to pyro. Will probably just leave for now.